controls/raddocking/how-to/add-menu-items-to-the-radpanes-menu.md

```diff
@@ -14,136 +14,220 @@ The purpose of this tutorial is to show you how to customize the __RadPane__ men
       
 >tipFor more information about the __RadPane__ structure, read [here]({%slug raddocking-panes-radpane%}) or read [this one]({%slug raddocking-features-panes-panes-menu%}) if you want to learn more about the __RadPane's__ menu.
 
-## Customizing the RadPane's Menu
+## Customizing the RadPane's Menu 

```

Comment:
Customizing RadPane's Menu (without the)
We should also remove this from the Title of the article and add a description.